code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -44,15 +44,17 @@
 	var/payment_department = ACCOUNT_SEC
 
 /mob/living/simple_animal/bot/secbot/beepsky
-	name = "Officer Beep O'sky"
-	desc = "It's Officer Beep O'sky! Powered by a potato and a shot of whiskey."
+	name = "Commander Beep O'sky"

```

Comment:
beepsky is a much better name than beep o sky

Reply:
Beepsky is the nickname for Beep O'Sky

Reply:
beep o sky is a lame name and literally no one uses it